### PR TITLE
Fix for upload with params

### PR DESCRIPTION
**Pull request by kellyp**
Fix:
- sends the options parameters with the file when it is uploaded.

### CODE REVIEW

File: lib/config.go [hunk 1]
```diff
@@ -186,16 +184,11 @@ func (config Config) ProcessUrlFromConfig(commandName string, subcommandName str
...
-	url := subCommandConfig.Url
+	Debugf("url template: %s", subCommandConfig.Url)
 
-	templ := template.Must(template.New("url").Parse(url))
-	buf := bytes.Buffer{}
-	err = templ.Execute(&buf, params)
-	if err != nil {
-		return "", err
-	}
+	url := subCommandConfig.Url
```
Comment (dan9186) [inline lib/config.go]: Is it really even setting it into a variable?

Comment (kellyp) [inline lib/config.go]: geez, every time I try to be lazy!

File: lib/params.go [hunk 2]
```diff
@@ -92,7 +94,27 @@ type GetParams struct {
 }
 
 func (params GetParams) String() string {
-	return fmt.Sprintf("List=%s, Project=%s, Url=%s, Type=%s, Checksum=%s, Id=%s, Text=%s, Version=%s, Limit=%s, Offset=%s", params.List, params.Project, params.Url, params.Type, params.Checksum, params.Id, params.Text, params.Version, params.Limit, params.Offset)
+	urlValues, err := url.ParseQuery("")
```
Comment (dan9186) [inline lib/params.go]: This could be shortcut with `urlValues := url.Values{}`

Comment (dan9186) [inline lib/params.go]: And then no need for error checking an error that will never happen.

File: lib/config.go (hunk 1 above)
Comment (dan9186) [inline lib/config.go]: That was supposed to say "worth setting it into a variable", but fingers are hard to work.

**Comment by dan9186** [comment]
👍 LGTM, waiting on Travis

**Pull request merged**